plugins/processors/awsappsignals/processor.go

```diff
@@ -132,6 +138,10 @@ func (ap *awsappsignalsprocessor) processMetrics(ctx context.Context, md pmetric
 			metrics := ils.Metrics()
 			for k := 0; k < metrics.Len(); k++ {
 				m := metrics.At(k)
+				// Check if metric name is lowercase
```

Comment:
Can't we make all metric names start with a capital letter with `Title()` and not worry about the map?